jupyterlab/commands.py

```diff
@@ -1071,6 +1071,8 @@ def _get_extensions_in_dir(self, dname, core_data):
             path = osp.realpath(target)
             extensions[name] = dict(path=path,
                                     filename=osp.basename(path),
+                                    link=data.get('homepage',
+                                        data.get('repository')['url']),
```

Comment:
I do not know if `homepage` is mandatory so I return `repository` when `homepage` is not available.